aeneas/src/mach/RegState.v3

```diff
@@ -0,0 +1,115 @@
+// Copyright 2023 Virgil Authors. All rights reserved.
+// See LICENSE for details of Apache 2.0 license.
+
+class RegState(regSet: MachRegSet, codegen: SsaMachGen) {
```

Comment:
Can you add a few comments for the top-level class and each field to describe what they do?